chore(sandbox): remove sandbox-k8 folder

mayank0202 opened the pull request:
## Description

Removal of sandbox k8  and moved to [terraform-sourceloop-apps](https://github.com/sourcefuse/terraform-sourceloop-apps) repo

shubhamp-sf commented:
@mayank0202 please form the commit based on conventional commits.

Doing `npm install && npx lerna bootstrap` will do the husky installation for you.

Then you can commit it usually.

jamescrowley321 commented:
@mayank0202 Update the sandbox/readme since the components about k8s are now in another repo.

mayank0202 commented:
> @mayank0202 Update the sandbox/readme since the components about k8s are now in another repo.

Do we need to add build.sh file also over there? I have just added a line that we moved k8 folder in terraform sourceloop-apps but further there are some steps written for applying k8 so need to know if it should be there or not @jamescrowley321
